core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/request/ConceptMapMappingSearchRequest.java

```diff
@@ -44,8 +50,31 @@ protected ConceptMapMappings doExecute(BranchContext context) throws IOException
 				.put(SearchResourceRequest.OptionKey.SORT_BY, sortBy())
 				.build();
 		
-		return context.service(ConceptMapMappingSearchRequestEvaluator.class)
-				.evaluate(context.service(ResourceURI.class), context, options);
+		List<ConceptMapMappings> evaluatedMappings = context.service(RepositoryManager.class)
+			.repositories()
+			.stream()
```

Comment:
No need, we don't know which tooling provides actual implementation of `ConceptMapMappingSearchRequestEvaluator` interface, but we know they all should provide one, so it is possible to perform a search on all of them and it will have the correct results.

Reply:
Following discussion, the `toolingId` filter is used for a different purpose here.